express-pg-auth0/src/api/todos/entities/insert-todo.entity.ts

```diff
@@ -1,6 +1,4 @@
 import { Insert } from '@database/operations';
 import { Todo } from './todo.entity';
 
-export type InsertTodo = Insert<
```

Comment:
Why changing this type? The `Insert<>` operation is going through `NullableKeysPartial<>` which makes only the nullable keys partial, not every key. `Partial<>` does not represent the correct type as for an insert we'll always need `name` and `userId`